app/routes/photos/components/GalleryPictureModal.tsx

```diff
@@ -324,14 +323,16 @@ export default class GalleryPictureModal extends Component<Props, State> {
               </Dropdown>
             </Flex>
           </Content>
+
           <Flex className={styles.pictureContainer}>
             <ProgressiveImage
               key={picture.id}
               src={picture.file}
-              alt="some alt"
```

Comment:
How was this approved

Reply:
After years of looking at our codebase, I can confirm, we used to have quite low standards.